src/core/Mini_rule.ml

```diff
@@ -48,6 +48,7 @@ type rule = {
    *)
   pattern_string : string;
   fix : string option;
+  fix_regexp : (string * int option * string) option;
```

Comment:
maybe introduce an intermediate type because those string * int option * string are a bit mysterious
maybe something in Rule.ml like type fix_regexp = {
   field1:
   field2:
  field3:
  }